stories/demos/components/RowRenderers/GroupRowRenderer.less

```diff
@@ -0,0 +1,15 @@
+.rdg-row-expand-icon {
+  cursor: pointer;
+
+  &:hover {
+    color: #777;
+  }
+}
+
+.rdg-row-default-group {
+  width: var(--row-width);
+  height: var(--row-height);
+  line-height: var(--row-height);
+  padding: 0 8px;
+  border-bottom: 1px solid #ddd;
```

Comment:
`border-bottom` seems like a common variable, I'm wondering if we want to create a css variable for this.